products/coldfusion.md

```diff
@@ -41,22 +41,22 @@ releases:
     releaseDate: 2025-04-08
```

Comment:
```suggestion
    releaseDate: 2025-02-25
```

Took this opportunity to fix the release date too (see https://community.adobe.com/t5/coldfusion-discussions/now-live-adobe-coldfusion-2025-release/td-p/15176421).